.firebase/hosting.ZGlzdA.cache

```diff
@@ -0,0 +1,8 @@
+index.html,1676615037473,36f32018f46239a8bdb06b76ddcf441ea901f5063a9566845203e9c7a6f463ad
+css/app.f8abe6a5.css,1676615037473,2a7bc2b0037743a04f0de82cbccb0952be5187943e4f4359ed978dd7c1886f96
+favicon.ico,1676615037473,1e71457865f706dc865b49a54a86e193818220d290b30226b6630a42faf1535d
+js/app.3afe4756.js,1676615037473,69da15b9fa0b949cfa014edec6efe02683e7822a21e53aad7c2c1b46cd2fe7b7
+js/app.3afe4756.js.map,1676615037473,2db355311f429d247986d0bbe36675093a37a5ca907b69f2d8ce6b7173901690
+js/chunk-vendors.6fb1622b.js,1676615037473,d51bfcb949fd2c189b6bdc0b77a6d52a41d4fde9512b85495b8cbfdf0649d637
+css/chunk-vendors.49870ecd.css,1676615037473,ee0bed076cd50a92abfc9e84dd7093e26b5be5d4d40994cf9b83e268f5eb34e8
+js/chunk-vendors.6fb1622b.js.map,1676615037473,c4ea504ba72939326ef1f0945c6146723e81b6cd5d76f48d4f4a7a75adbf38a9
```

Comment:
pls create a `.gitignore` file to the project and add `.firebase` to it.

This will stop commits from checking in anything in this folder to the repo as it's not important